### ISSUE TITLE
Feature Request: print connection_info JSON 

### ISSUE THREAD

**Issue by jacksund** [spyder-ide/spyder-kernels#358]
Hello! This is a feature-request, but I hope opening an issue is fine.

Is it possible to have connection info automatically printed when `python -m spyder_kernels.console` is called?

Based on the [ipython recipe](https://github.com/ipython/ipython/wiki/Cookbook:-Connecting-to-a-remote-kernel-via-ssh#connect_info), it might be possible by having spyder call `%connect_info` as soon as the kernal is started. I know very little about kernels though.

As an extra, it'd be much easier to post the printed json directly into the dialog here. It'd be an alternative to the select file option shown here:
![image](https://user-images.githubusercontent.com/47992949/147867090-f485c39b-ee20-4154-9c35-b914870c630a.png)

This would make your [remote kernel tutorial](http://docs.spyder-ide.org/current/panes/ipythonconsole.html?highlight=ssh#connect-to-a-remote-kernel) much easier.

Let me know what your team thinks!

**Comment by jacksund** [comment]
As an update, I wrote a simple class based on `fabric` to make this process easier for me. The code might be useful if your team decides to implement these features. You can find that code [here](https://github.com/jacksund/simmate/blob/dev/src/simmate/configuration/ssh.py).

**Comment by ccordoba12** [comment]
This is a very interesting idea and would simplify the process to connect to remote kernels a lot. Thanks for the suggestion!

We'll try to implement it in a future release.

**Comment by matifali** [comment]
Any updates?

**Comment by ccordoba12** [comment]
No, sorry. We've been asking for funding to work on this for last two years but we haven't gotten it yet.

**Comment by matifali** [comment]
Now, most IDEs provide a direct and simpler way to connect to a remote server over ssh,

1. VS code through remote VS code extension

2. JetBrains through Gateway

3. Spyder is the only one missing.

**Comment by dhirschfeld** [comment]
> *3. Spyder is the only one missing*

https://opencollective.com/spyder
